source/release-notes.txt

```diff
@@ -20,6 +20,30 @@ Release Notes for |bi|
 .. expect and we don't break the links on the downloads page.
 .. (DOCS-9670 for context)
 
+.. _bi-2-14-7:
+
+|bi| 2.14.7
```

Comment:
I am not too sure where you got this?
Here is the link to the release notes : https://jira.mongodb.org/secure/ReleaseNote.jspa?projectId=13184&version=36193
There is nothing really visible for users. The 2 tasks are internal.
We rebuilt the driver with Go 1.19.9+ because there were security vulnerabilities with the previous version.
So, I think you might just want to say : Now built with GO 1.19.9+

Reply:
I think the JIRA ticket may have linked to the wrong release notes. I just addressed your feedback - please take a look if you get a chance. Thank you for your help!

Reply:
ah, that might explains things :)